app/routes.py

```diff
@@ -1,2 +1,62 @@
-from flask import Blueprint
+from app import db
+from app.models.solar import Planet
+from flask import Blueprint,jsonify, make_response, request
 
+
+planets_bp = Blueprint("planets_py", __name__, url_prefix= "/planets")
+
+@planets_bp.route("", methods=["GET","POST"])
+def handle_planets():
+    if request.method == "GET":
+        planets = Planet.query.all()
+        planets_response = []
+        for planet in planets:
+            planets_response.append(
+                {
+                    "id": planet.id,
+                    "name": planet.name,
+                    "description": planet.description
+                })
```

Comment:
We could reduce these lines of code using a helper method to turn the planet object into a dictionary in the Planets class:

```suggestion
    planets_response = []
    for planet in planets:
        planets_response.append(planet.to_dict())
```
And further refactor using a list comprehension :D 

```suggestion
        planets_response = [ planets_response.append(planet.to_dict()) for planet in planets ]

```